### DISCUSSION TITLE
Is it safe to use __experimentalBlockVariationPicker and the other apis that go along with it?

### DISCUSSION THREAD

**Discussion by xdrew760x** [WordPress/gutenberg#54037] [Q&A] [answered]
I Would like to create a block using this API however i'm unsure if it's safe to do so?

**Comment by cheestudio** [comment]
@xdrew760x did you ever get an answer about this? I'm looking to do the same.

**Reply by tyxla** [reply, marked as answer]
@xdrew760x @cheestudio `__experimentalBlockVariationPicker` has been part of the WordPress core for a few years now, and for that time, its API hasn't really changed. That means that it's implicitly stable, and using it for your projects should be safe. 

You might want to keep an eye out for the `__experimental` name deprecation in the future, in which case it will renamed as part of the stable APIs (`BlockVariationPicker`). When that happens, both `__experimentalBlockVariationPicker` and `BlockVariationPicker` will be valid usages for some time, giving y'all time to migrate your imports from `__experimentalBlockVariationPicker` to `BlockVariationPicker`.

Hope that helps, but if you have any other questions, let me know!

**Reply by xdrew760x** [reply]
Thank you for the response.
